src/utils/planner.ts

```diff
@@ -19,7 +19,7 @@ export const planner = {
    * @param id Planner ID.
    */
   async getPlanById(id: string): Promise<PlannerPlan> {
-    const requestOptions = getRequestOptions(`${graphResource}/v1.0/planner/plans/${id}`, 'none');
+    const requestOptions = getRequestOptions(`${graphResource}/v1.0/planner/plans/${id}`, 'minimal');
```

Comment:
Rather than always retrieving metadata, we should extend the `getPlanById` signature with an extra option that allows you to specify the metadata level and keep `none` as the default value. That way, we don't introduce overhead to commands that don't need metadata.